index.js

```diff
@@ -302,6 +302,9 @@ class K8sExecutor extends Executor {
                     waitingReason !== 'StartError')
             );
         };
+        this.dlcEnabled = hoek.reach(options, 'kubernetes.dlcEnabled', { default: false });
+        this.dlcPath = hoek.reach(options, 'kubernetes.dlcPath');
+        this.dlcMountPath = hoek.reach(options, 'kubernetes.dlcMountPath');
```

Comment:
no defaults?

Reply:
ah nvm I see them in buildcluster-queue-worker

Reply:
yeah, defaults would be empty, i'll add that